### PR TITLE
Create keypair

### LINKED ISSUE #4: Generate public/private key
**Feature description**
Generation public/private keys pair 

- [x] Create APDU call(request, method) for getting the public key #6 
- [x] Add python script for testing the APDU call and documentation how does it work #7  #8 

Part of #3

### PR DESCRIPTION
**Pull request by Retsediv**
The whole functionality for public/private keys and address generation. User interface with easy-to-use menu and screen with address. 

Fixes #4 #9 #7 #6 #16 

## Proposed Changes

  -  Keypair generation
  -  Address generation
  -  APDU call to get public key  
  -  UI (menu, screen with scrolling address)
  -  Using BIP44 now 

## Screenshots:
Main menu:
image

Inside settings:
image

In no address generated yet:
photo_2019-02-22_17-58-26

Display actual address:
image

### CODE REVIEW

**Review by gz-c — COMMENTED** [review]
* What value is used for the `coin_type` part of the bip44 path?
* Does ledger have a base58 encoding API?

**Comment by Retsediv** [comment]
@gz-c 
> * What value is used for the `coin_type` part of the bip44 path?

Just a dummy data for now. (`80000378` in hex)

> * Does ledger have a base58 encoding API?

No, there is no such function in BOLOS SDK.

File: scripts/get_public_key.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import binascii
+
+from ledgerblue.comm import getDongle
+from ledgerblue.commException import CommException
+
+
+bipp32_path = ( # example of bip32 path
```
Comment (olemis) [inline scripts/get_public_key.py]: `bip32_path`

File: scripts/get_public_key.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      "8000002C"
+    + "80000378"
+    + "80000000"
+    + "00000000"
+    + "00000000"
```
Comment (olemis) [inline scripts/get_public_key.py:12]: Is the final value meant to be `"8000002C80000378800000000000000000000000"` ?

```sh

$ python -c 'v = (
> "8000002C"
>     + "80000378"
>     + "80000000"
>     + "00000000"
>     + "00000000"
> ); print(v)'
8000002C80000378800000000000000000000000
```

**Review by olemis — CHANGES_REQUESTED** [review]

File: scripts/get_public_key.py (hunk 2 above)
Comment (Retsediv) [reply scripts/get_public_key.py:12]: Yes
I have divided into pieces to see parts of bip path clearly

File: scripts/get_public_key.py (hunk 1 above)
Comment (Retsediv) [reply scripts/get_public_key.py]: oh, that's a typo(
have just fixed

**Pull request merged**